.travis.yml

```diff
@@ -1,13 +1,16 @@
 # Run tests run on travis-ci after each commit and before each merge
 
 language: java
+dist: trusty
```

Comment:
Giving other failures because of this. Will try to fix it but if unable then can't build installer in Travis.